__tests__/edit.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import MockStorage from '../src/__mocks__/local-storage.js';
+import MockDOM from '../src/__mocks__/DOM.js';
+
+describe('Edit Function', () => {
+  const temp = [
+    {
+      description: 'Sample 1',
+      completed: false,
+      index: 0,
+    },
+    {
+      description: 'Sample 2',
+      completed: false,
+      index: 1,
+    },
+    {
+      description: 'Sample 3',
+      completed: false,
+      index: 2,
+    },
+  ];
```

Comment:
- [ ] Please Add a test that really tests the function not an array of objects, and I do see that you are repeating the same thing for the `completed.test.js` + `clearAll.test.js`.